broker/bpmn.go

```diff
@@ -99,24 +111,63 @@ func addEmitHandlers(state *bpmn.State) {
 	state.AddTaskHandler("putUser", updateUserAndAgency)
 }
 
+//	======================================
+//	LEAD FUNCTIONS
+//	======================================
+
+func addLeadHandlers(state *bpmn.State) {
+	state.AddTaskHandler("setLeadData", setLeadBpmn)
+	state.AddTaskHandler("sendLeadMail", sendLeadMail)
+}
+
+func setLeadBpmn(state *bpmn.State) error {
+	policy := state.Data
+	setLeadData(policy)
+	return nil
+}
+
+func sendLeadMail(state *bpmn.State) error {
+	policy := state.Data
+	log.Printf(
+		"[sendProposalMail] from '%s', to '%s', cc '%s'",
+		fromAddress.String(),
+		toAddress.String(),
+		ccAddress.String(),
+	)
+	mail.SendMailLead(
+		*policy,
+		fromAddress,
+		toAddress,
+		ccAddress,
+	)
+	return nil
+}
+
+//	======================================
+//	PROPOSAL FUNCTIONS
+//	======================================
+
 func addProposalHandlers(state *bpmn.State) {
 	state.AddTaskHandler("setProposalData", setProposalBpm)
```

Comment:
Remember that who decides what handlers to run is the config in function-data, so make sure that there is no more references to "sendProposalMail" in any broker-flow